src/Telegram.Bot/TelegramBotClient.cs

```diff
@@ -664,7 +756,8 @@ public Task<User> GetMeAsync(CancellationToken cancellationToken = default)
             IEnumerable<InputMediaBase> media,
             bool disableNotification = default,
             int replyToMessageId = default,
-            CancellationToken cancellationToken = default
+            CancellationToken cancellationToken = default,
+            bool allowSendingWithoutReply = default
```

Comment:
The default convention in .NET world is to setup `CancellationToken` at the end of the calling params.